src/olympia/lib/settings_base.py

```diff
@@ -1473,8 +1473,7 @@ def read_only_mode(env):
 SITEMAP_DEBUG_AVAILABLE = False
 
 CINDER_SERVER_URL = env(
-    'CINDER_SERVER_URL',
-    default='https://stage.cinder.nonprod.webservices.mozgcp.net/api/v1/',
+    'CINDER_SERVER_URL', default='https://mozilla-staging.cinderapp.com/api/v1/'
```

Comment:
It's not clear to me if this is the right approach, and if it is should it default to staging or production.

I would imagine we actually want `settings_base` to not have this value defined by default. and let the dev/stage/prod configs set it as well as the `settings.py` we use for local dev to set it to stage.

This could lead to traffic in production going to cinder staging if we somehow remove the override in the production config.. that could be very bad and difficult to detect immediately.

whereas if the value is simply undefined.. all kinds of stuff will just break immediately (in this case the preferable path)

wdyt?